src/main/java/org/opensearch/neuralsearch/processor/chunker/DelimiterChunker.java

```diff
@@ -21,7 +23,7 @@ public DelimiterChunker(Map<String, Object> parameters) {
 
     public static final String DEFAULT_DELIMITER = ".";
 
-    private String delimiter = DEFAULT_DELIMITER;
+    private String delimiter;
```

Comment:
Why not assign default value like fixed token length algorithm?

Reply:
It will be assigned inside the validate function.